akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/ActorTestKitScalaTestSpec.scala

```diff
@@ -52,5 +46,11 @@ abstract class ActorTestKitWordSpec(testKit: ActorTestKit) extends ActorTestKitB
    */
   implicit val patience: PatienceConfig =
     PatienceConfig(testKit.testKitSettings.DefaultTimeout.duration, Span(100, org.scalatest.time.Millis))
+
+  // FIXME document to call super if override
+  override protected def afterAll(): Unit = {
```

Comment:
I think this was missing, i didn't see where it was shutdown

Reply:
indeed, added it as abstract in ActorTestKitBase but forgot our own impl of it